components/TransferDialog/index.tsx

```diff
@@ -0,0 +1,115 @@
+import styled from 'styled-components'
+import { Dialog } from '../Dialog'
+import { Text } from '../Text'
+import { WalletCardWithInput } from './WalletCardWithInput'
+import { WalletCardWithBalance } from './WalletCardWithBalance'
+import { Button } from '../Button'
+import { useTokenInfo } from '../../hooks/useTokenInfo'
+import { useState } from 'react'
+import { TransactionKind } from './types'
+
+type TransferDialogProps = {
+  tokenSymbol: string
+  transactionKind: TransactionKind
+  isShowing: boolean
+  onRequestClose: () => void
+}
+
+export const TransferDialog = ({
+  tokenSymbol,
+  transactionKind,
+  isShowing,
+  onRequestClose,
+}: TransferDialogProps) => {
+  const capitalizedTransactionType =
+    transactionKind === 'deposit' ? 'Deposit' : 'Withdraw'
+
+  const tokenInfo = useTokenInfo(tokenSymbol)
+
+  const [tokenAmount, setTokenAmount] = useState(0)
+  const tokenMaxAvailableBalance = 1000
+  const walletAddressTransferringAssetsFrom =
+    'cosmos1uw6ls6y8du6d1uw6ls6y8du6d1uw6ls6y'
```

Comment:
here's the wallet address we rendered from which the transaction is going to be executed. if the asset is being transferred from the platform, then that's the Juno wallet address, if that's from the wallet, then wallet address thereby